### DISCUSSION TITLE
disable pagination [Jan 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4868] [Discord Threads]
> 🙋‍♂️ **noviscera**  
## Is it possible to turn off pagination? What a pain  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1051031226112299029/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### i think not. why would u disable the pagination?  
#  
> 🙋‍♂️ **noviscera**  
#### so you can scroll and see everything rather than try and figure out what page a particular record is on?  
#  
> 🙋‍♂️ **noviscera**  
#### if it were separated alphabetically that would almost be fine  
#  
> 🙋‍♂️ **noviscera**  
#### like an old-style rolodex. but we've got 500+ entries  
#  
> 🙋‍♂️ **noviscera**  
#### and it's important to us to be able to sort across multiple categories dynamically, so a static order isn't necessarily suitable either  
#  
> 🙋‍♂️ **wkw**  
#### so it s more like infinite scroll  
#  
> 🙋‍♂️ **wkw**  
#### cuz potentially the table could hv large dataset. loading all in one go causes performance issue. so your case s more like scroll to fetch pages but display in the same page. am i right?  
#  
> 🙋‍♂️ **noviscera**  
#### yeah, that'd be fine!  
#  
> 🙋‍♂️ **noviscera**  
#### I can see why it's not on by default, if there are 50,000 records instead of 500  
#  
> 🙋‍♂️ **wkw**  
#### someone requested that before. u may subscribe this thread:   
#### https://github.com/nocodb/nocodb/issues/3568  
#  
> 🙋‍♂️ **noviscera**  
#### yess. thank you!  
#  
> 🙋‍♂️ **noviscera**  
#### wish I had enough money to sponsor! this is my first experience with a project of this type and I'm really keen for it all to work properly  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1051031226422669312)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
